src/sentry/api/endpoints/organization_request_project_creation.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework import serializers
+
+
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.bases import OrganizationPermission
+from sentry.api.serializers.rest_framework import CamelSnakeSerializer
+from sentry.utils.email import MessageBuilder
+from sentry.utils.http import absolute_uri
+
+
+class OrganizationRequestProjectCreationSerializer(CamelSnakeSerializer):
+    target_user_email = serializers.EmailField(required=True)
+
+
+class OrganizationRequestProjectCreationPermission(OrganizationPermission):
```

Comment:
Hmm, do we have a permission that already matches this?

Reply:
Like do we have to define this class, or is there something in api.bases we can use. Looks like maybe not

Reply:
@EvanPurkhiser Yea, I'm sure we do some places but generally `POST` endpoints require some sort of write permission.